### ISSUE TITLE
Prisma Adapter + Mysql + Unable to SignIn

### ISSUE THREAD

**Issue by sudoak** [nextauthjs/next-auth#3822]

**Comment by ndom91** [comment]
`nextauthjs/next-auth` has been migrated to a monorepository. The adapters code can now be found there under `packages/adapter-*`. Thanks for your interest in the project!

**Comment by ndom91** [comment]
Hi @sudoak can you please provide us with some more detailed information? Like your `[...nextauth].js` configuration? Thanks!

**Comment by sudoak** [comment]
![Screenshot 2022-02-05 222923](https://user-images.githubusercontent.com/25544768/152651060-54015936-5824-4523-afa0-57ba3c12edc6.png)

![Screenshot 2022-02-05 223006](https://user-images.githubusercontent.com/25544768/152651064-158dd1ea-d7d0-456d-842e-ad48762ffffa.png)

**Comment by ndom91** [comment]
> Hi @ndom91, I followed your above mentioned steps, still getting error.
> 
> ![Screenshot 2022-02-05 222923](https://user-images.githubusercontent.com/25544768/152651060-54015936-5824-4523-afa0-57ba3c12edc6.png)
> 
> ![Screenshot 2022-02-05 223006](https://user-images.githubusercontent.com/25544768/152651064-158dd1ea-d7d0-456d-842e-ad48762ffffa.png)
> 

Yeah now you have a different error, your trying to put too much info in the `id_token` and it becomes too long for your database to save with the configured field type. 

It seems your using MySQL, right? Prisma seems to map this `String` type from the schema to `varchar(191)` in MySQL under the hood (see: https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#string). 

You may have to either manually change this `id_token` field type to something dynamic or longer, or try to save less in it.
